website/src/types/model.ts

```diff
@@ -1,8 +1,8 @@
-// docs.jsonの型
-// cf. docs/src/model.rs
+// Type definitions for `docs.json`.
+// Reference: `docs/src/model.rs`
 
 /**
- * ページ情報
+ * Page information.
```

Comment:
```suggestion
 * Details about a documentation page and its children.
```

(copied from the original `docs/src/model.rs`)